src/include/OpenImageIO/fmath.h

```diff
@@ -1229,6 +1229,7 @@ convert_type (const S &src)
 /// shifted fully to the right.
 template<unsigned int FROM_BITS, unsigned int TO_BITS>
 inline OIIO_HOSTDEVICE unsigned int bit_range_convert(unsigned int in) {
+    static_assert(FROM_BITS > 0, "FROM_BITS cannot be 0");
```

Comment:
The static_assert makes it impossible to successfullyinstantiate the template with FROM_BITS == 0. So why is the `if (FROM_BITS)` necessary later on?

Reply:
There's 2 variations of bit_range_convert (1 template and 1 normal) and I suspect you're looking at the wrong one from the diff?

Reply:
Oh, right you are. My brain thought I was just looking later in the same template. My bad.